src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -335,10 +335,12 @@ protected ResponseEntity searchUsersByOrganisation(String organisationIdentifier
         return responseEntity;
     }
 
-    protected ResponseEntity<ModifyUserRolesResponse> modifyRolesForUserOfOrganisation(UserProfileUpdatedData userProfileUpdatedData, String userId, Optional<String> origin) {
+    protected ResponseEntity<ModifyUserRolesResponse> modifyRolesForUserOfOrganisation(UserProfileUpdatedData userProfileUpdatedData, String userId, Optional<String> origin, String orgId) {
 
         userProfileUpdatedData = userProfileUpdateRequestValidator.validateRequest(userProfileUpdatedData);
```

Comment:
Will this validate Org id for external endpoint ? For ext we don't need.

Reply:
I have addressed this in the following commit: https://github.com/hmcts/rd-professional-api/pull/444/commits/96a95e934368f752c84eea079929984454a543a0